test/conftest.py

```diff
@@ -392,54 +381,39 @@ def pudl_tmpdir(tmp_path_factory):
     return tmpdir
 
 
-@pytest.fixture(scope="session")
-def pudl_output_tmpdir(pudl_tmpdir):
-    tmpdir = pudl_tmpdir / "output"
-    tmpdir.mkdir()
-    return tmpdir
-
-
-@pytest.fixture(scope="session")
-def pudl_input_tmpdir(pudl_tmpdir):
-    tmpdir = pudl_tmpdir / "data"
-    tmpdir.mkdir()
-    return tmpdir
+def pytest_sessionstart(session):
+    """Configures input/output paths for the tests."""
+    # TODO(rousik): Should we be using fixed paths instead
```

Comment:
It seems like we're setting up the `PudlPaths` here and also in `pudl_path_setup`.

In theory, we want the *inputs* to default to whatever our env var points to, because we don't want to have to redownload a bunch of data. And then we want the *outputs* to go to a tmpdir so we don't pollute our dev outputs - which is what the `pudl_path_setup` function does better than `pytest_sessionstart`.

But that only works when `pudl_path_setup` is explicitly requested - should we just pull that logic into `pytest_sessionstart`?

Reply:
Okay. I have switched this to session scoped `configure_paths_for_tests` fixture that has `autouse=True` which essentially executes that before any tests regardless of whether it is explicitly requested or not. That should cover all the cases.